### PR TITLE
Feature/movements history modules tests

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/vue/modules/movements-history/store/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+      id: 'BDPFDXJAL6UY53L52NNWPD7RTAO4EVZL55SWHNYVYJQ44BOEIQKL4FOJ',
+      asset: {
+        id: 'BTC',
+        type: 'assets',
+      },
+    },
+  ]
+  const movements = [
+    {
+      id: '176093659138',
```
Comment (quinn71) [inline src/vue/modules/movements-history/store/index.spec.js]: JSON API requires both `type` and `id` present in resource. It's better to follow the spec even in mocks

Comment (Datikzz) [reply src/vue/modules/movements-history/store/index.spec.js]: Added type

File: src/vue/modules/movements-history/store/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+import { Wallet } from '@tokend/js-sdk'
+
+import { Movement } from '../wrappers/movement'
+import { Balance } from '../wrappers/balance'
+
+import * as Api from '../_api'
+import accountBalancesJSON from '@/test/mocks/account-balances'
+
+describe('movements-history.module', () => {
+  const accountId = 'GDIU5OQPAFPNBP75FQKMJTWSUKHTQTBTHXZWIZQR4DG4QRVJFPML6TTJ'
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/vue/modules/movements-history/store/index.spec.js]: Declared variables inside test-cases

File: src/vue/modules/movements-history/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+              .resolves()
+
+            await wrapper.vm.loadMovementsFirstPage()
+
+            expect(wrapper.vm.isMovementsLoaded).to.be.true
+
+            wrapper.vm.loadMovements.restore()
+          })
+
+        it('handles the error if loading was failed', async () => {
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/vue/modules/movements-history/index.spec.js]: Splitted to 2 tests

File: src/vue/modules/movements-history/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
+import MovementsHistoryModule from './index'
+import { movementsHistoryModule } from './store/index'
+
+import Vuex from 'vuex'
+
+import { createLocalVue, shallowMount } from '@vue/test-utils'
+import { Wallet } from '@tokend/js-sdk'
+
+import * as Api from './_api'
```
[the remark this answers is not in the document]
Comment (Datikzz) [reply src/vue/modules/movements-history/index.spec.js]: Renamed api to ApiImporter

**Pull request merged**